modules/openy_gc_auth/openy_gc_auth.info.yml

```diff
@@ -5,3 +5,5 @@ package: Open Y Virtual YMCA
 version: 0.1
 core: 8.x
 core_version_requirement: ^8 || ^9
+dependencies:
```

Comment:
It should work only if user enabled this module.
We have to keep everything working if openy_gc_log was disabled.